plugins/jsconnect/class.jsconnect.plugin.php

```diff
@@ -24,6 +24,15 @@ class JsConnectPlugin extends Gdn_Plugin {
 
     /// METHODS ///
 
+    public function addControl($key, $item) {
```

Comment:
Hark, a new method with no docblock. I name thee Scoundrel and bite my thumb at Thou.